features/vesting/vestingDetailedSlide.tsx

```diff
@@ -125,14 +128,14 @@ export const VestingDetailedSlide: FC<VestingDetailedSlideProps> = memo(
             <Column $primary>
               <DetailsHeader>Available</DetailsHeader>
               <DetailsValue>
-                <FormatToken amount={unclaimed} symbol={token?.symbol} />
+                <FormatToken amount={unclaimed} symbol={token?.symbol || ''} />
```

Comment:
Same here, it should allow to pass undefined

Reply:
needs patch to FormatToken (warehouse)